rust/src/metadata.rs

```diff
@@ -25,7 +25,8 @@ use serde::{Deserialize, Serialize};
 
 use crate::Logger;
 
-const METADATA_FILE: &str = "selenium-manager.json";
+const METADATA_FILE_OLD: &str = "selenium-manager.json";
```

Comment:
I think we should just change this or throw a warning saying it is changed and remove it next release.

Reply:
The metadata file is entirely transparent for the user since it is handled automatically by SM. This change does not support any impact on the use of SM, so we should not display any warning about it. A warning trace will alert to something that does not deserve the user's attention. On the other hand, yes, we can remove the logic for `METADATA_FILE_OLD` in the next release.

Reply:
I'm confused if it isn't for backwards compatibility for the user, why do we need it for this release? 😄

Reply:
The motivation and context of this change is explained at the beginning, as usual.

Reply:
I get the motivation for changing the name, I don't get the motivation for copying things over from the old file. Building a new metadata file for a new release seems minimal difficulty? I get there could end up being an old metadata file in the directory for current users, but if it isn't for users that doesn't seem like a big issue, either. Dunno, just trying to minimize the amount of code that we know will get removed later.

Reply:
Nothing is copied from the old file. Three lines of code check the existence of the old metadata file, and if it exists, it is deleted. It is a way to get the cache cleaned (i.e., without the former metadata file, which is not used anymore). As I said before, these three lines can be deleted in upcoming releases.